internal/pkg/scorecard/scorecard.go

```diff
@@ -230,48 +232,82 @@ func ScorecardTests(cmd *cobra.Command, args []string) error {
 				}
 			}()
 		}
-		if err := createFromYAMLFile(viper.GetString(GlobalManifestOpt)); err != nil {
-			return fmt.Errorf("failed to create global resources: %v", err)
-		}
-		if err := createFromYAMLFile(viper.GetString(NamespacedManifestOpt)); err != nil {
-			return fmt.Errorf("failed to create namespaced resources: %v", err)
-		}
 	}
 
-	if err := createFromYAMLFile(viper.GetString(CRManifestOpt)); err != nil {
-		return fmt.Errorf("failed to create cr resource: %v", err)
-	}
-	obj, err := yamlToUnstructured(viper.GetString(CRManifestOpt))
-	if err != nil {
-		return fmt.Errorf("failed to decode custom resource manifest into object: %s", err)
+	crs := viper.GetStringSlice(CRManifestOpt)
+	// check if there are duplicate CRs
+	combinedManifests := []byte{}
+	for _, cr := range crs {
+		file, err := ioutil.ReadFile(cr)
+		if err != nil {
+			return fmt.Errorf("failed to read file: %s", cr)
+		}
+		combinedManifests = yamlutil.CombineManifests(combinedManifests, file)
 	}
-	if err := waitUntilCRStatusExists(obj); err != nil {
-		return fmt.Errorf("failed waiting to check if CR status exists: %v", err)
+	gvks, err := getGVKs(combinedManifests)
```

Comment:
Is there a reason we need to combine the manifests and then basically undo that to get each CR GVK? Can we pass each file into `getGVKs` and build the full list here?